vowpalwabbit/example.h

```diff
@@ -80,6 +117,16 @@ struct example : public example_predict  // core example datatype.
   // input fields
   polylabel l;
 
+  // Notes: TLDR; needed to make predict() independent of label (as it should theoretically should be)
```

Comment:
do we need this comment to be committed to the code base? now that `initial` is not part of label it seems like the comment is just keeping history around (explanation of this change could be a comment on the PR)